specs/e2e/flows.spec.ts

```diff
@@ -0,0 +1,111 @@
+import { getSalesChannelToken } from "@commercelayer/js-auth"
+
+import { test, expect } from "../fixtures/tokenizedPage"
+
+const clientId = process.env.NEXT_PUBLIC_CLIENT_ID as string
+const endpoint = process.env.NEXT_PUBLIC_ENDPOINT as string
+const organizationSlug = new URL(endpoint).hostname.split(".")[0]
+const scope = process.env.NEXT_PUBLIC_MARKET_ID as string
+const skuListId = process.env.E2E_SKU_LIST_ID as string
+
+const makeRemoteUrl = async ({
+  cart,
+  all,
+  inline,
+}: {
+  cart: boolean
+  all: boolean
+  inline: boolean
+}) => {
+  const auth = await getSalesChannelToken({
+    clientId,
+    endpoint,
+    scope,
+  })
+  const token = auth?.accessToken as string
+  return `https://${organizationSlug}.commercelayer.app/microstore/list/${skuListId}?accessToken=${token}&cart=${cart}&all=${all}&inline=${inline}`
+}
+
+test("hosted flow: microstore > cart > checkout", async ({ page }) => {
+  const remoteUrl = await makeRemoteUrl({
```

Comment:
I would use the tokenizedPage and open the local version of the microstore, not the production one. Then for cart and checkout applications we can use the production one. If you want we can discuss it on a quick call